app/scripts/controllers/network/network-controller.js

```diff
@@ -202,26 +202,9 @@ export default class NetworkController extends EventEmitter {
       this.emit(NETWORK_EVENTS.INFURA_IS_UNBLOCKED);
     }
 
-    let networkVersion;
-    let networkVersionError;
-    try {
-      networkVersion = await this._getNetworkId();
```

Comment:
This isn't right. We should still make this call so we can double check that the network is accessible. See: https://github.com/MetaMask/core/pull/1073/files#diff-ea0aa37793484e278bd59970b7a6ce8380b75b723b34612081d402fb66261cfdR335